client/my-sites/themes/controller.js

```diff
@@ -68,6 +69,19 @@ function makeElement( ThemesComponent, Head, store, props ) {
 	);
 };
 
+// Where do we put this? It's client/server-agnostic, so not in client/controller,
+// which requires ReactDom... Maybe in lib/react-helpers?
```

Comment:
Once `client/` is renamed to `app/`, the fact that we can't put this helper in `app/controller` will be even more awkward, so why not instead work on making sure that _it's okay_ to have the server require a module that requires `ReactDom`?